uui-components/src/buttons/Button.tsx

```diff
@@ -15,9 +13,16 @@ export interface ButtonProps
 
     /** CSS classes to put on the caption */
     captionCX?: CX;
+
+    /** CountIndicator component */
+    countIndicator?: React.ComponentType<IHasCaption> ;
+
+    /** Indicator property, default = false */
+    indicator?: boolean;
```

Comment:
We will fix it in future with a new component.

Reply:
With which one? If you mean status component, it's a little bit different, because will not have outline border and horizontal paddings. 
Anyway for badge component, should have indicator by design, and i would like that it will be only in bages, not in interface of Button.

Reply:
removed it from button in Badge styles via :before